test/fixtures/output/kafka/quota/delete-prompt.golden

```diff
@@ -1,2 +1,2 @@
-Are you sure you want to delete client quota "cq-123"?
-To confirm, type "cq-123". To cancel, press Ctrl-C: Deleted client quota "cq-123".
+Are you sure you want to delete client quota "cq-1234"?
```

Comment:
This was a mismatch between the handler (`cq-1234`) and the tests (`cq-123`), so I just picked one.